Content.Server/Resist/CanEscapeInventoryComponent.cs

```diff
@@ -11,6 +12,20 @@ public sealed partial class CanEscapeInventoryComponent : Component
     [DataField("baseResistTime")]
     public float BaseResistTime = 5f;
 
+    /// <summary>
+    ///     Initial amount of time when you cannot be picked up when dropped while escaping
+    /// </summary>
+    [DataField("basePenaltyTime")]
```

Comment:
```suggestion
    [DataField]
```
DataField names get automatically converted to camelCase nowadays. Old files still do this manually, but we don't need it anymore. You can do the same with the other DataField above.